server/handler.go

```diff
@@ -163,7 +164,7 @@ func (h *Handler) handleDocImpl(w http.ResponseWriter, req *http.Request, signUR
 			return
 		}
 		if xerrors.Is(err, fetch.ErrURLMismatch) {
-			replyClientError(w, err)
+			log.Print("Warning: %w", err)
```

Comment:
1. I think this still prints the string "error" to stdout? By virtue of the definition of `err.Error()` at: https://github.com/google/webpackager/blob/c7597c58f5e83513153ff8b77ad2e7486bd2c597/error.go#L40
2. This never writes a reply to `w`; I'm not sure what the default HTTP response is in that case. I think you still want to call `replyError` to set up a 400 response for cases where somebody curls `/priv/doc/https://wrongurl.com/`.

Reply:
Done.